gems/sorbet-runtime/Rakefile

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 task :test do
-  Dir.glob('./test/**/*.rb').each(&method(:require))
+  Dir.glob('./test/**/*.rb').reject {|path| path.match(/^.\/test\/types\/sealed_module\//)}.each(&method(:require))
```

Comment:
can we make naming scheme more descriptive?
WDYT about looking `___no_autorequire__` in the path of the file